src/views/Rules/ListRules/index.jsx

```diff
@@ -215,6 +222,37 @@ function ListRules(props) {
       rowsPerPage={RULES_ROWS_PER_PAGE}
     />
   );
+  const handleRuleDelete = rule => {
+    setDialogState({
+      ...dialogState,
+      open: true,
+      title: 'Delete Rule?',
+      body: `This will schedule a delete for rule ${rule.rule_id}.`,
```

Comment:
This text isn't necessarily true. Currently, we're only directly deleting rules (no scheduled changes involved). I think eventually we'll delete directly when signoff isn't required, and delete via a scheduled change when it is.

I'm going to update this page to do that in then next day or two.

Reply:
Sounds good. I'll go ahead and remove "schedule" from the sentence.